docs/notebooks/qhsri-tutorial.pct.py

```diff
@@ -0,0 +1,232 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Batch HSRI Tutorial
+
+# %% [markdown]
+# Batch Hypervolume Sharpe Ratio Indicator (qHSRI) is a method for picking a batch of query points during Bayesian Optimisation. It makes use of the Sharpe Ratio, a portfolio selection method used in investment selection to carefully balance risk and reward.
```

Comment:
we should cite the reference paper upfront, 
Batch Hypervolume Sharpe Ratio Indicator (qHSRI) is a method **proposed by ....** for picking